### ISSUE TITLE
Impossible to navigate to editable from toolbar by keyboard and vice versa

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor4#438]
## Are you reporting a feature or a bug?

Bug

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)

http://dev.ckeditor.com/ticket/13168

## Provide detailed reproduction steps (if any)

1. Open editor page perhaps http://sdk.ckeditor.com/samples/accessibility.html.
1. Focus on editable.
1. Press Alt + F10 to navigate to toolbar.
1. Press Alt + F11 to navigate to elements path.

### Expected result

It should be possible to navigate to elements path right from toolbar and vice versa.

### Actual result

You can't navigate to elements path.

**Comment by Comandeer** [comment]
We can slightly enhance the initial fix, using design proposed in https://dev.ckeditor.com/ticket/13168#comment:13

**Comment by jacekbogdanski** [comment]
Let's see if https://github.com/ckeditor/ckeditor4/pull/2412 solves that issue.

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5252
